website/docs/r/cluster.html.markdown

```diff
@@ -348,6 +348,13 @@ In addition to all arguments above, the following attributes are exported:
 * `mongo_uri` - Base connection string for the cluster. Atlas only displays this field after the cluster is operational, not while it builds the cluster.
 * `mongo_uri_updated` - Lists when the connection string was last updated. The connection string changes, for example, if you change a replica set to a sharded cluster.
 * `mongo_uri_with_options` - connection string for connecting to the Atlas cluster. Includes the replicaSet, ssl, and authSource query parameters in the connection string with values appropriate for the cluster.
+* `connection_strings` - Set of connection strings that your applications use to connect to this cluster.
+    - `connection_strings.standard` -  `mongodb://` A required prefix to identify that this is a string in the standard connection format.
+    - `connection_strings.standard_srv` - The `mongodb+srv` protocol tells the driver to look up the seed list of hosts in DNS.
```

Comment:
- `connection_strings.standard_srv` - Public mongodb+srv:// connection string for this cluster.  The mongodb+srv protocol tells the driver to look up the seed list of hosts in DNS. Atlas synchronizes this list with the nodes in a cluster. If the connection string uses this URI format, you don’t need to append the seed list or change the URI if the nodes change. Use this URI format if your driver supports it. If it doesn’t, use connectionStrings.standard.

Reply:
changes has been made, thank you melissa.